### ISSUE TITLE
Enable user selection of date format in time/date settings

### ISSUE THREAD

**Issue by GitRASQ** [budgie-desktop/budgie-desktop#1039]
The clock on the panel can show the date in the format 'MM/DD/YYYY'. I want to use ISO format but I don't appear to have any way to change the format. Time and Date Settings should allow the user to pick a date format, either with a combo box or a format string like the `date` command uses.

**Operating System**: Solus
**Budgie version**: Budgie 10.3.1

**Comment by orschiro** [comment]
By ISO date format you basically mean this `2017-07-17`?

**Comment by GitRASQ** [comment]
Yes, I want the panel's clock widget to show that and as far as I can tell that's not currently possible.

**Comment by orschiro** [comment]
Doesn't date formatting depend on the LOCALE settings?

**Comment by GitRASQ** [comment]
Yes, it seems changing my locale in lightdm affects the date format (only after rebooting for some reason). This is a good way to choose a default date format, but the user should still be able to override it.

**Comment by orschiro** [comment]
Well, you can change it by choosing a different locale. Why isn't this sufficient?

**Comment by GitRASQ** [comment]
So I used google to find out which locale uses ISO format, went to lightdm to tell it that I'm actually from Denmark, scrolled through a giant list of locales for nearly a minute before finding it, rebooted, remembered that for some reason lightdm doesn't save my settings unless I login after changing them, selected the locale again, logged in, rebooted again, then logged in again. Now I have my date in ISO format. 

Even if I knew I could do this in the first place (and I didn't), its... not exactly a user friendly method. And I'd assume locale affects more than just date format which I don't really want. So no, I don't think this is sufficient.
